### PR TITLE
Update .NET Tools page to 5.x

### LINKED ISSUE #370: .NET CLI Page Too Large?
https://github.com/aspnet/EntityFramework.Docs/blob/live/entity-framework/core/miscellaneous/cli/dotnet.md

I feel like the documentation here should be broken up into more pages, thoughts?

- Installation Page
- Usage: EF-Database Page
- Usage: EF-Context Page
- Usage: EF-Migrations Page

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 263f21d6-6edc-7582-e0e2-fa1c9670bcc0
* Version Independent ID: b3dd8c84-18b1-f98d-64f1-7897d1482692
* Content: [EF Core tools reference (.NET CLI) - EF Core](https://docs.microsoft.com/en-us/ef/core/miscellaneous/cli/dotnet)
* Content Source: [entity-framework/core/miscellaneous/cli/dotnet.md](https://github.com/dotnet/EntityFramework.Docs/blob/master/entity-framework/core/miscellaneous/cli/dotnet.md)
* Product: **entity-framework**
* Technology: **entity-framework-core**
* GitHub Login: @bricelam
* Microsoft Alias: **bricelam**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #370
[Internal review URL](https://review.docs.microsoft.com/en-us/ef/core/miscellaneous/cli/dotnet?branch=pr-en-us-2650)

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@ErikEJ please review. Let me know if you'd like a public view of the  rendered doc.

**Comment by ErikEJ** [comment]
@Rick-Anderson I think you mean to ping @bricelam  ?

File: entity-framework/core/miscellaneous/cli/dotnet.md [hunk 1]
```diff
@@ -3,40 +3,40 @@ title: EF Core tools reference (.NET CLI) - EF Core
...
 
-If you're using Visual Studio, we recommend the [Package Manager Console tools](xref:core/miscellaneous/cli/powershell) instead:
+When using Visual Studio, consider using the [Package Manager Console tools](xref:core/miscellaneous/cli/powershell) in lieu of the CLI tools. Package Manager Console tools automatically:
 
-* They automatically work with the current project selected in the **Package Manager Console** without requiring that you manually switch directories.
-* They automatically open files generated by a command after the command is completed.
+* Works with the current project selected in the **Package Manager Console** without requiring that you manually switch directories.
+* Opens files generated by a command after the command is completed.
+
+CLI commands should ***not*** be used in the Package Manager Console.
```
Comment (bricelam) [inline entity-framework/core/miscellaneous/cli/dotnet.md:19]: I don't agree with this statement. You can use them, you just have to remember to `cd` into project directories and specify `--startup-project`

File: entity-framework/core/miscellaneous/cli/dotnet.md [hunk 2]
```diff
@@ -3,40 +3,40 @@ title: EF Core tools reference (.NET CLI) - EF Core
...
 
-If you're using Visual Studio, we recommend the [Package Manager Console tools](xref:core/miscellaneous/cli/powershell) instead:
+When using Visual Studio, consider using the [Package Manager Console tools](xref:core/miscellaneous/cli/powershell) in lieu of the CLI tools. Package Manager Console tools automatically:
 
-* They automatically work with the current project selected in the **Package Manager Console** without requiring that you manually switch directories.
-* They automatically open files generated by a command after the command is completed.
+* Works with the current project selected in the **Package Manager Console** without requiring that you manually switch directories.
+* Opens files generated by a command after the command is completed.
```
Comment (bricelam) [inline entity-framework/core/miscellaneous/cli/dotnet.md:17]: You also get tab completion of commands, parameters, project names, context types, and migration names.

File: entity-framework/core/miscellaneous/cli/dotnet.md [hunk 3]
```diff
@@ -61,42 +61,6 @@ The `dotnet ef` commands are included in the .NET Core SDK, but to enable the co
...
-      <TargetFramework>netcoreapp1.1</TargetFramework>
-    </PropertyGroup>
-    <ItemGroup>
-      <PackageReference Include="Microsoft.EntityFrameworkCore.Design"
-                        Version="1.1.6"
-                         PrivateAssets="All" />
-    </ItemGroup>
-    <ItemGroup>
-       <DotNetCliToolReference Include="Microsoft.EntityFrameworkCore.Tools.DotNet"
-                              Version="1.1.6" />
```
Comment (bricelam) [inline entity-framework/core/miscellaneous/cli/dotnet.md:93]: 🔥🔥🔥

**Review by bricelam — APPROVED** [review]

File: entity-framework/core/miscellaneous/cli/dotnet.md (hunk 1 above)
Comment (Rick-Anderson) [reply entity-framework/core/miscellaneous/cli/dotnet.md]: @bricelam  thanks I addressed both of your comments.

**Comment by Rick-Anderson** [comment]
@bricelam  Merge when you're happy with this.

**Pull request merged**